process/block/metablock.go

```diff
@@ -1242,7 +1242,7 @@ func (mp *metaProcessor) CommitBlock(
 	lastHeader := mp.blockChain.GetCurrentBlockHeader()
 	lastMetaBlock, ok := lastHeader.(data.MetaHeaderHandler)
 	if !ok {
-		log.Warn("metaBlock.CommitBlock - nil current block header")
+		log.Debug("metaBlock.CommitBlock - nil current block header")
```

Comment:
conflicting feelings about this one
if not at genesis then it should be a warn

Reply:
changed